admin-dev/themes/new-theme/js/components/module-card.ts

```diff
@@ -450,7 +453,8 @@ export default class ModuleCard {
           BOEvent.emitEvent('Module Installed', 'CustomEvent', mainElement);
         };
 
-        jqElementObj.replaceWith(result[moduleTechName].action_menu_html);
+        jqElementObj = $(result[moduleTechName].action_menu_html).replaceAll(jqElementObj);
```

Comment:
Maybe it's worth a comment here
```suggestion
        // Since we replace the DOM content we need to update the jquery object reference to target the new content, and we need to hide the new content which is not hidden by default
        jqElementObj = $(result[moduleTechName].action_menu_html).replaceAll(jqElementObj);
```